db/data/20220520103603_add_new_allocation_subject_general_sciences.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class AddNewAllocationSubjectGeneralSciences < ActiveRecord::Migration[6.1]
+  def up
+    allocation_subject = AllocationSubjects::GENERAL_SCIENCES
```

Comment:
We should call this variable `allocation_subject_name` as we create the actual `AllocationSubject` further down. By that logic `subject_specialisms` should also be `subject_specialism_names`